packages/lib/src/components/ApplePay/ApplePay.tsx

```diff
@@ -35,7 +46,7 @@ class ApplePayElement extends UIElement<ApplePayConfiguration> {
     /**
      * Formats the component props
      */
-    protected formatProps(props) {
+    protected override formatProps(props) {
```

Comment:
It might be worth in the future fixing this everywhere in new PR, and adding `noImplicitOverride` to TS config.

Reply:
Yea. That would be nice. 
Actually I think we should start removing the disabled strict features that we have on `tsconfig.json` until the point the we get the 'strict: true' mode . That will make us have a more reliable codebase